UPGRADE.md

```diff
@@ -84,15 +84,11 @@ The non-interface methods of driver-level classes have been marked internal:
 - `OCI8Connection::getExecuteMode()`
 - `OCI8Statement::convertPositionalToNamedPlaceholders()`
 
-## Deprecated `DBALException`
-
-The `Doctrine\DBAL\DBALException` class has been deprecated in favor of `Doctrine\DBAL\Exception`.
-
 ## Inconsistently and ambiguously named driver-level classes are deprecated
 
 The following classes under the `Driver` namespace have been deprecated in favor of their consistently named counterparts:
 
-- `DriverException` → `Exception`
+- `DriverException` → `LowLevelDriverException`
```

Comment:
This is insane.

Reply:
I am not happy with it myself, the existence of `DriverException` in both Exception and Driver namespaces is a bit confusing, and its annoying that an alias is necessary.

Reply:
I am happy with what we have right now since the current naming doesn't have this ambiguity. The `Exception` class is a generic exception at a given level of abstraction (the global PHP `\Exception`, the library-level exception, and the driver-level exception). The `DriverException` at the library level means that this exception comes from the driver and has corresponding properties.

> its annoying that an alias is necessary

**You're fighting the wrong problem**. First of all, its not necessary, you can use a partially-qualified name if needed. Second, is a natural side effect of using namespaces.

Reply:
Reverted to using `Driver\DriverException` including the alisases in `DBALException` and `Exception\DriverException`.